docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+services:
+  web:
+    build:
+      context: .
+      target: dev
+    init: true
+    command: django-admin runserver 0.0.0.0:8000
+    environment:
+      SECRET_KEY: fake
+      DJANGO_SETTINGS_MODULE: wagtailio.settings.dev
+      DATABASE_URL: postgres://wagtailio:wagtailio@db:5432/wagtailio
```

Comment:
nitpicky nitpick: it would've been nice to steer away from ..io in the light of wagtail.io -> wagtail.org move